iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="sizedImgDiv" role="img" aria-label$="{{alt}}" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" alt$="{{alt}}" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
You can't directly programmatically inspect the accessibility tree just yet - there are a couple of experimental APIs around but not really in any usable state for your purposes.

The Accessibility Developer Tools does have some utils for trying to work out the computed name and role of a particular element:

https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/AccessibilityUtils.js#L626
https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/Properties.js#L236

Obviously in this case you'd still need to look at the child nodes inside shadow DOM, though.